### DISCUSSION TITLE
Webcam Error

### DISCUSSION THREAD

**Discussion by Tsavvis** [merkourisa/Augmented-Reality#21] [Q&A]
Καλησπέρα, όσο δούλευα για το δεύτερο παραδοτέο, παρατήρησα ότι ό,τι αλλαγή και αν κάνω στον κωδικά μου, εμφανίζεται το εξής μήνυμα:
![εικόνα](https://user-images.githubusercontent.com/100925413/167894415-bd36fc18-25d2-4fa6-8a8b-bf17e8caf526.png)
Να σημειωθεί ότι από χθες που δούλευα τον ίδιο κώδικα, δεν εμφανιζόταν τέτοιο θέμα.

**Comment by merkourisa** [comment]
Καλησπέρα @Tsavvis, δοκίμασε κάποιον υπολογιστή του εργαστηρίου αλλά πρόσεξε να είναι συνδεδεμένη η κάμερα ή αν έχεις ανεβάσει την εργασία σε github pages δοκίμασε να ανοίξεις την εφαρμογή από το κινητό σου

**Reply by Tsavvis** [reply]
Τελικά δοκίμασα από κινητό και δουλεύει κανονικά. Ευχαριστώ
